### PR TITLE
Change if(len(args) == 0)  to if(args) for readability

### PR DESCRIPTION
**Pull request by zkoppert**
Signed-off-by: Zack Koppert <zachery.koppert@tektronix.com>

### Pull Request Checklist

Please review the [Contribution Guidelines](../README.md#contributing) before submitting.

- [x] Pulling against the `dev` branch (left side).
- [x] You have previously submitted a Contributor License Agreement or have contacted a maintainer to request one.

### Type (Select only one)

- [x] Bug fix
- [ ] Working example
- [ ] In-progress example (need Keithley help)
- [ ] In-progress example (do not need Keithley help)

### Description

Fix for #9

### CODE REVIEW

File: Drivers/DMM6500_DAQ6510/DMM6500_Python_Sockets_Driver/DMM6500_Sockets_Driver.py [hunk 1]
```diff
@@ -246,12 +246,12 @@ def SetFunction_DC_Voltage(self, *args):
         #   For channel scan measurements:
         #       1. Channel string
         #       2. Input impedance
-        if (len(args) == 0):
+        if (args):
```
Comment (nlynchjo) [inline Drivers/DMM6500_DAQ6510/DMM6500_Python_Sockets_Driver/DMM6500_Sockets_Driver.py]: would `if not args:` work here??

Comment (zkoppert) [reply Drivers/DMM6500_DAQ6510/DMM6500_Python_Sockets_Driver/DMM6500_Sockets_Driver.py]: I believe you are correct. I have commit the changed to applicable areas. Can you scan through my changes again for logical consistency?

**Comment by zkoppert** [comment]
> The changes at lines 249, 275, 286, and 339 appear to be the inverse of their intended operation, which is "if there are no arguments, then do this thing..". This is evidenced by the change requested at line 254 (and others) where we expect there to be arguments passed.
> 
> If this is just a matter of me not yet fully understanding the outcome of the usage then it may be fine. However, I would feel more comfortable seeing this tested for usability before approving.

@jbrown1234  How can I go about testing these changes? I believe they are now ready to be checked out.

**Comment by nlynchjo** [comment]
Any chance this has been tested?  Okay to merge?

**Pull request merged**